### ISSUE TITLE
Seems to have trouble with large fasta files

### ISSUE THREAD

**Issue by EReckase** [lmdu/pyfastx#1]
Wanted to benchmark this against pyfaidx for getting subsequences, but I can't get pyfastx to work with my hg19 genome. pyfasta indexes the file ok, and then says it has 28 sequences, but when I try to access subsequences, I get either empty strings or crashes.

```
>>> f = pyfastx.Fasta('hg19.fa')
>>> f.fetch('chr1', (100000,110000))
''
>>> f.fetch('chr1', (1000000,1100000))
SystemError: Objects/longobject.c:244: bad argument to internal function
>>> s = f[-1]
SystemError: Objects/longobject.c:244: bad argument to internal function
>>> f.keys()
<Identifier> contains 28 identifiers
```

```
[~]# pyfastx info hg19.fa
Sequence counts: 28
Total bases: 3095694378
GC content: 40.90%
A counts: 844868179
T counts: 846097379
C counts: 585018013
G counts: 585360526
N counts: 234350281
Mean length: 110560513.50
Median length: 111259709.00
Max length: 249250621
Min length: 131
N50, L50: 155270560, 8
length >= 1000: 25
```

This is Python 2.7 running on CentOS. This is version 0.5.6 of pyfastx.

**Comment by lmdu** [comment]
This problem was caused by separated int and long type in Python 2.7. Because there is only long type in Python3. I only checked the long type of user input and ignored the int type in Python 2.7. I have fixed it in new version 0.5.7. Thank you for reporting this issue.

**Comment by EReckase** [comment]
Thank you!
